Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml

```diff
@@ -38,16 +38,60 @@
   - type: StaticPrice
     price: 500
 
-- type: entity
+- type: entity #Frontier
```

Comment:
Suggesting commenting as shown below
```yaml
# Frontier - Mosin rework (PR#1391)
- type: entity
...
malus: 0.225
# End Frontier
```

Reply:
That actually sounds like a much better way to do it then from what I heard how it was done before.